packages/core-util/package.json

```diff
@@ -51,6 +51,7 @@
   "devDependencies": {
     "@aws-sdk/types": "^3.696.0",
     "@iot-app-kit/eslint-config": "*",
+    "@iot-app-kit/helpers": "*",
```

Comment:
should this be a devDependency? do we need it just for local dev or for runtime as well?

Reply:
Updated!